build-system/tasks/check-links.js

```diff
@@ -128,12 +111,12 @@ function isLinkToFileIntroducedByPR(link) {
 }
 
 /**
- * Filters out whitelisted links before running the link checker.
+ * Filters out links in allow-list before running the link checker.
  *
  * @param {string} markdown Original markdown.
- * @return {string} Markdown after filtering out whitelisted links.
+ * @return {string} Markdown after filtering out allowed links.
  */
-function filterWhitelistedLinks(markdown) {
+function filterAllowedLinks(markdown) {
```

Comment:
The latest [API docs](https://github.com/tcort/markdown-link-check#markdownlinkcheckmarkdown-opts-callback) of the link checker appear to have introduced an `ignorePatterns` option that lets you specify a list of regexes to ignore. I think it's cleaner than modifying the markdown text before passing it to the checker. Will do this in a follow up PR. Nothing to do now.

Reply:
Done in #25235.